students/YingGuo/session04/mailroom2.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+#session04, mailroom2
+
+from collections import defaultdict
+data = defaultdict(list)
+Sample = {"Tom":[100,200], "Tim":[3000], "Ken": [888,888,888], "Ted":[8000,5000,500], "Jon":[1000,6000]}
+for key in Sample.keys():
+    data[key] = Sample[key]
+
+#Send a thank you function 
+def thank_you(data):
+	while True:
+		user_input = input("please enter the name of donor. You can enter list to see the exisitng list of donors. Or enter quit to exist current function")
+		if user_input.lower() == "list":
+			print(data.keys())
+
+		elif user_input.lower() == "quit":
+			break
+		
+		else:
+			amount1 = int(input("how much would you like to donat?"))
+			data[user_input.capitalize()] += [amount1]
+			print("Thank you {} for your donation {}!".format(user_input.capitalize(), amount1))
+			return data
+			break
```

Comment:
Minor: No need to `break`, because it can never be reached (because previous line is `return`).